client/my-sites/domains/map-domain/index.jsx

```diff
@@ -126,8 +124,10 @@ export class MapDomain extends Component {
 				} );
 			return;
 		} else if ( this.props.isSiteOnPaidPlan ) {
-			wpcom
-				.addDomainMapping( selectedSite.ID, domain )
+			wpcom.req
+				.post( `/sites/${ selectedSite.ID }/add-domain-mapping`, {
+					domain,
+				} )
```

Comment:
just a formatting nit: could the `{ domain }` object be an one-liner, like the one on line 113 is? Or does Prettier break it anyway?

Reply:
Sure, updated in https://github.com/Automattic/wp-calypso/pull/57912/commits/2ac24a6a62beaab70b6c7849a060e3e2349b9ecd.